snapshot

```diff
@@ -3,30 +3,32 @@
 # This script takes snapshots of the files/data directory on google cloud instances
 
 # To setup as a cron job:
-# 0 9 * * * /var/lib/spida/snapshot.sh $disk-id
+# 0 9 * * * /var/lib/spida/snapshot.sh $DISK_NAME $EMAIL $CLIENT_NAME $ENVIRONMENT $PRODUCT $ZONE
```

Comment:
is this the name of the script when installed?

Reply:
just realized that's an old directory, but the create.sh script uses the file name disk-snapshot.sh instead of snapshot.sh when storing the script on the instance

i'm going to just remove the directory from here and keep it as snapshot.sh